package.json

```diff
@@ -21,12 +21,13 @@
     "@rsksmart/rsk-testnet-contract-metadata": "^1.0.3",
     "axios": "^0.21.0",
     "bn.js": "^5.1.3",
-    "did-jwt-vc": "^1.0.7",
     "did-resolver": "^2.1.1",
+    "ethereumjs-util": "^7.0.7",
     "ethjs-contract": "^0.2.3",
     "ethjs-query": "^0.3.8",
     "ethr-did-resolver": "^3.0.1",
     "identicon.js": "^2.3.3",
+    "jesse-did-jwt": "^4.6.2",
```

Comment:
We should create a version in rsksmart

Reply:
hahaha, yes. I need this for both the Id Manager and the Verifier. [This PR in the did-jwt](https://github.com/rsksmart/did-jwt/pull/1) package is what I used. However, it needed to explicitly set `"did-resolver": "2.1.1"`.

Reply:
Let's merge it and open an issue for updating this dependency